src/gpu/intel/jit/gemm/generator/pieces/common.cxx

```diff
@@ -597,7 +598,7 @@ void BLASKernelGenerator<hw>::calcIncrement(LDIncrements &increments, Subregiste
 
 // Get a multiple of lda/ldb for address increments.
 template <HW hw>
-SubregisterPair BLASKernelGenerator<hw>::lookupIncrement(const LDIncrements &increments, const SubregisterPair &base, int scale, const CommonStrategy &strategy, CommonState &state, bool *release)
+SubregisterPair BLASKernelGenerator<hw>::lookupIncrement(const LDIncrements &increments, const SubregisterPair &base, int scale, bool isA64, const CommonStrategy &strategy, CommonState &state, bool *release)
```

Comment:
Let's store `isA64` inside of `LDIncrements` so it doesn't need to be passed in here.